### PR TITLE
fix: show full description instead of fragment

### PR DESCRIPTION
**Pull request by geeky-abhishek**

### CODE REVIEW

File: src/shared/invite-response/parts/invite-reply-part.tsx [hunk 1]
```diff
@@ -83,41 +85,56 @@ const InviteReplyPart: FC<InviteReplyPart> = ({
...
+						label="NO"
+						icon="Close"
+						color="error"
+						onClick={onAction('DECLINE')}
+						disabled={participationStatus === 'DE'}
+					/>
+				</Padding>
+				<Padding horizontal="small" top="medium">
+					<Button
+						label="PROPOSE NEW TIME"
```
Comment (zovomat) [inline src/shared/invite-response/parts/invite-reply-part.tsx]: these need to be translated

Comment (nubsthead) [inline src/shared/invite-response/parts/invite-reply-part.tsx]: missing translation key

**Review by nubsthead — CHANGES_REQUESTED** [review]
There is an issue with appointments description css.
image

**Review by dhavaldodiya — DISMISSED** [review]

**Review by giuliano176 — COMMENTED** [review]
I did test on a random invite and the fragment is not rendering 
in my test BodyMessageRenderer is not composed correctly, fullinvite.textDescription is undefined

**Comment by geeky-abhishek** [comment]
> I did test on a random invite and the fragment is not rendering in my test BodyMessageRenderer is not composed correctly, fullinvite.textDescription is undefined Screenshot_20220221_114838

@giuliano176 It's because it's not an invite.You can check on zimbra also XD

**Review by nubsthead — DISMISSED** [review]

**Review by zovomat — DISMISSED** [review]

**Review by zovomat — APPROVED** [review]

**Review by nubsthead — APPROVED** [review]

**Pull request merged**